cleanup/cleanup.sh

```diff
@@ -1,3 +1,25 @@
-#!/bin/sh
+#!/bin/bash
 
-echo "Plugin called."
+set -e
```

Comment:
Add
* `-u` - undefined vars are errors
* `-o pipefail` - Abort immediately if a command that is part of a pipe fails